CHANGELOG.md

```diff
@@ -2,10 +2,14 @@
 
 ## Unreleased
 
+Adds
+
+* Allow project-level developer to override bundling decisions by configuring the `@apostrophecms/asset` module.
```

Comment:
using the new `rebundleModules` option. [Link to the reference page for the asset module, which will be updated around release time to cover this]